$NetBSD: patch-ab,v 1.3 2011/10/17 08:47:51 obache Exp $

* LOG_AUTHPRIV is not in Solaris, use LOG_AUTH instead.

--- pam_af.c.orig	2011-08-24 07:53:32.000000000 +0000
+++ pam_af.c
@@ -41,6 +41,9 @@
 #include <fcntl.h>
 #include <time.h>
 #include <syslog.h>
+#ifndef LOG_AUTHPRIV
+#define LOG_AUTHPRIV LOG_AUTH
+#endif
 #include <assert.h>
 #include <ndbm.h>
 
